src/current/_includes/v23.1/misc/table-storage-parameters.md

```diff
@@ -1,6 +1,7 @@
 |                    Parameter name                    |                                                                    Description                                                                    | Data type | Default value |
 |------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|-----------|---------------|
 | `exclude_data_from_backup`                           | Exclude the data in this table from any future backups.                                                                                           | Boolean   | `false`       |
+| <span class="version-tag">New in v23.1.15:</span> `schema_locked` | Disallow schema changes on this table. Enabling `schema_locked` can help [improve performance of changefeeds]({% link {{ page.version.version }}/create-changefeed.md %}#disallow-schema-changes-on-tables-to-improve-changefeed-performance) running on this table. | Boolean | `false` |
```

Comment:
the mention of 'schema changes' could link to schema changes docs for this version eg https://www.cockroachlabs.com/docs/v23.1/online-schema-changes